app/controllers/admins/orders_controller.rb

```diff
@@ -22,5 +22,18 @@ def deliver
         redirect_to admins_orders_url, notice: "Order wasn't delivered"
       end
     end
+
+    def prepare_to_pick_up
+      @order = Order.find(params[:id])
+      begin
+        ActiveRecord::Base.transaction do
+          EmployeeMailer.reward_pick_up_instruction_email(@order).deliver
```

Comment:
mailers can be send outside transaction - also we should send email after we prepare_to_pick_up

Reply:
done ✅